### ISSUE TITLE
is there a public serviceUrl I can test with?

### ISSUE THREAD

**Issue by ralyodio** [samelm/minimal-lib-jitsi-meet-example#1]
I donj't know what to put here. I'm just trying to prototype and see if I can get jitsi lib working w/o having to host a backend.

**Comment by samelm** [comment]
@ralyodio 
Hi, you can use official Jitsi url `//meet.jit.si/http-bind` or `wss://meet.jit.si/xmpp-websocket` for test

**Comment by ralyodio** [comment]
any reason this wonj't work on http://localhost? it works fine on https in production

**Comment by samelm** [comment]
@ralyodio
Hi, I fixed example
Now you can `npm run start` and then connect to test room https://meet.jit.si/minimal-lib-jitsi-meet-example-room

**Comment by ralyodio** [comment]
I am trying with this and i get a connection error.

```
const options = {
	openBridgeChannel: true,
	//serviceUrl: "//meet.jit.si/http-bind" /* PASTE YOUR OWN */,
	serviceUrl: 'https://meet.jit.si/http-bind',
	hosts: {
		domain: 'meet.jitsi',
		muc: 'muc.meet.jitsi',
		focus: 'focus.meet.jitsi'
	},
	useStunTurn: true,
	p2p: {
		enabled: false,
		useStunTurn: true
	}
};

```

My video shows but the other persons does not

**Comment by samelm** [comment]
Please, use new main.js version, correct connection urls are there

**Comment by ralyodio** [comment]
That worked but ios safari doesn't show video.

**Comment by samelm** [comment]
Yes, it is possible, I remember there were problems with iOS safari
Try iOS chrome
You can also read the official Jitsi forum about browser support
